pkg/chunk/purger/purgeplan/delete_plan.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package purgeplan;
+
+option go_package = "purgeplan";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/cortexproject/cortex/pkg/ingester/client/cortex.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+message Plan {
+  int64 start_timestamp_ms = 1;
```

Comment:
You use both camel case and snake case in this file. I think for Protos we generally use snake_case across the board. Feel free to choose either, just make it consistent across the file.